src/python/pants/rules/core/fmt_test.py

```diff
@@ -20,15 +29,26 @@ class FmtTest(TestBase):
 
   @staticmethod
   def make_hydrated_target(
-    *, name: str = "target", adaptor_type: Type[TargetAdaptor] = PythonTargetAdaptor
+    *,
+    name: str = "target",
+    adaptor_type: Type[TargetAdaptor] = PythonTargetAdaptor,
+    include_sources: bool = True,
   ) -> HydratedTarget:
+    mocked_snapshot = Snapshot(
+      directory_digest=EMPTY_DIRECTORY_DIGEST,
```

Comment:
I agree with this. We also should probably factor out the entire function `make_hydrated_target`. We're now using a similar inlined implementation in (I believe) four different test files. I've been punting on generalizing this into a util function because I think it's too early - we're discovering what the API should look like. For example, it wasn't apparent we needed to use `EagerFileSet` when mocking a `TargetAdaptor` until this PR.

Going to punt on this but add a TODO.